src/stringProcessing/relevantWords.js

```diff
@@ -308,9 +309,43 @@ function getRelevantWords( text, locale ) {
 	return take( combinations, relevantWordLimit );
 }
 
+/**
+ * Gets relevant words from keyphrase and synonyms, metadescription, and subheadings.
+ *
+ * @param {string} keyphrase The keyphrase of the paper.
+ * @param {string} synonyms The synonyms of the paper.
+ * @param {string} metadescription The metadescription of the paper.
+ * @param {string[]} subheadings The subheadings of the paper.
+ * @param {string} locale The locale of the paper.
+ *
+ * @returns {WordCombination[]} Relevant word combinations from the paper attributes.
+ */
+function getRelevantWordsFromTopic( keyphrase, synonyms, metadescription, subheadings, locale ) {
+	let language = getLanguage( locale );
+	if ( ! functionWordLists.hasOwnProperty( language ) ) {
+		language = "en";
+	}
+
+	const functionWords = functionWordLists[ language ].all;
+
+	const subheadingsJoined = subheadings.join( " " );
+	const attributes = keyphrase.concat( " ", synonyms, " ", metadescription, " ", subheadingsJoined );
+	const attributesWords = uniq( getWords( attributes.toLocaleLowerCase() ) );
+
+	const attributesWordsFiltered = filter( attributesWords, function( word ) {
```

Comment:
This is a bit more concise and does not use lodash functions:
```js
const attributesWordsFiltered = attributesWords.filter( word => ! functionWords.includes( word.trim() ) );
```

Reply:
I always use this source to find the array methods I can use, instead of lodash functions:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
(Although lodash has some nice functions as well that plain old javascript arrays do not have).